curriculum/js1/9.js

```diff
@@ -4,8 +4,9 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, multipleOf7 = a + 1) => {
```

Comment:
i think multipleof7 isn't a good variable name because not all numbers are multiples of 7